docs/reference/supported-environments.md

```diff
@@ -4,6 +4,15 @@ title: "Supported environments"
 description: "Find out where to run Camunda 8 components for SaaS and Self-Managed, including Optimize for both Camunda 8 and Camunda 7."
 ---
 
+:::note
```

Comment:
https://github.com/camunda/camunda-docs/pull/2841#pullrequestreview-1711162341

Still not against these changes, but I feel like it's quite possible that it doesn't give us enough flexibility as per my above comment. Will this note also need extended if we have to drop support for _some_ of the patch releases of a supported minor version?

Reply:
@RomanJRW does this occur outside of Optimize? Regardless, instead of extending the note, we would explicitly mention what is not supported, patch or otherwise, in the table. 

Would you be more comfortable if it said the following instead (changing "will be compatible" to "should be compatible"):

> Higher or more recent versions should be compatible with Camunda, with respect to semver, unless otherwise noted.

Reply:
This issue can occur globally I would say. If there is a security issue that impacts one component, it will typically impact all of them. It happened with ES versions 7.10.x and also 7.17.x, so is not super common but also something we should consider. Happy to proceed as is and we can consider this scenario when it next happens

I think "will be" is better than "should be". We should document things like this with confidence imo 👍

Reply:
> there were security issues (from memory, I haven't gone back to check the specific reasons) that only got patched with 7.16.3. Specifying the patch version in the docs allowed us to be explicit in saying that that 7.16.1-2 weren't supported but that 7.16.3+ were, because the older versions contained a vulnerability that could potentially affect our applications.

@RomanJRW How have we been impacted by this? Was it a support case? Shouldn't we ask, the customer to run the latest patch before we start supporting them? As there any reason for a customer not to install the latest patch?

Reply:
https://jira.camunda.com/browse/OPT-5824
We handled it for 7.16.2, and it related to the two support cases linked there

It's not about the latest patch, it's about the minimum patch. The scenario I am trying to highlight is that it would be wrong for us to say we support 8.9+, for example, when not _all_ patches of that version are actually compatible